apps/judicial-system/scheduler/src/app/app.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from '@island.is/nest/config'
+
+export const appModuleConfig = defineConfig({
+  name: 'AppModule',
+  load: (env) => ({
+    backendAccessToken: env.required(
+      'BACKEND_ACCESS_TOKEN',
+      'secret-backend-api-token',
+    ),
+    backendUrl: env.required('BACKEND_URL', 'http://localhost:3344'),
+    timeToLiveMinutes: env.requiredJSON('TIME_TO_Live_MINUTES', 1),
```

Comment:
Should this be all caps?
```suggestion
    timeToLiveMinutes: env.requiredJSON('TIME_TO_LIVE_MINUTES', 1),
```